adapters/repos/db/vector/hnsw/config_update.go

```diff
@@ -58,18 +62,18 @@ func ValidateUserConfigUpdate(initial, updated schema.VectorIndexConfig) error {
 	return nil
 }
 
-type immutableInt struct {
-	accessor func(c ent.UserConfig) int
+type immutableParameter struct {
```

Comment:
why this struct was changed from `immutableInt` to `immutableParameter`? is it used somewhere else?

Reply:
`distance` is of type string, which is why the accessor now returns an `interface{}` instead on an int. `immutableParameter` is therefore a more fitting name. And no this is not used anywhere else to my knowledge.